templates/default/sql/dml/0.10.0.sql.erb

```diff
@@ -39,3 +39,4 @@ REPLACE INTO `hopsworks`.`variables` VALUES ("hive_scratchdir_cleaner_interval",
 
 REPLACE INTO `hopsworks`.`variables` VALUES ("admin_email", "<%= node['hopsworks']['admin']['email'] %>");
 
+REPLACE INTO `hopsworks`.`variables` VALUES ("localhost", "<%= node['install']['localhost'] %>");
```

Comment:
What's the purpose of this setting? There is no PR in Hopsworks that uses it. 
If it's not used, let's just get rid of it.

Reply:
I wanted to do a URL rewrite in the YarnProxyServlet. If 'localhost' is set, rewrite the target URL for the spark ui to localhost. This would relieve the need to rewrite /etc/hosts to point the fqdn to localhost.

Reply:
This `REPLACE` needs now to be moved to `1.0.0`